### ISSUE TITLE
Can not Flatten CoulombMatrix

### ISSUE THREAD

**Issue by geoffreyweal** [SINGROUP/dscribe#145]
Hello 

I see that in commit [6c1dfde](https://github.com/SINGROUP/dscribe/commit/6c1dfde885ea6fe6c375320ed3fefef675f166ed) removed the flattening option in descriptor such as the Coulomb Matrix. However, in the past I have been able to pass the CM into an AverageKernel, but it fails as it requires a flatten Coulomb matrix. 

I looked in the [dscribe manual for flattening](https://singroup.github.io/dscribe/latest/tutorials/descriptors/coulomb_matrix.html#flattening), and it says to use the flatten parameter to flatten the matrix, but this has not been removed.

What is the currently recommended way to flatten a descriptor matrix such as the coulomb matrix?

Thanks

**Comment by geoffreyweal** [comment]
So I was doing some more testing, and realised that the CM matrix is flattened automatically by dscribe. Is this done by design? If someone wanted the unflattened version, is there an automatic way of doing this, or you would need to manually unflatten it?

```
cm_monomer.reshape(n_atoms_max, n_atoms_max).T
```

What is the recommended approach for this? Thanks!

**Comment by lauri-codes** [comment]
Yes, I think starting from v2.0.0 onwards the descriptor output is flattened automatically always for all descriptors. If you want the original matrix, you can use: `coulomb_matrix.unflatten(features)`.

**Comment by geoffreyweal** [comment]
Excellent. Thanks for this recommendation! Appreciate it!
